src/servers/src/auth.rs

```diff
@@ -108,6 +108,12 @@ pub enum Error {
     #[snafu(display("Encounter IO error, source: {}", source))]
     IOErr { source: std::io::Error },
 
+    #[snafu(display("Encounter system error, msg: {}", msg))]
+    SystemErr { msg: String },
+
+    #[snafu(display("Rpc call return failed, {}: {}", code, msg))]
+    RpcResultErr { code: u32, msg: String },
```

Comment:
to avoid adding every error into this enum, we can add generic error type like `AuthBackendErr` with a `source` field to carry actual error.